jdl/converters/types.ts

```diff
@@ -30,3 +30,12 @@ export type JSONEntity = {
   fields?: JSONField[];
   relationships?: JSONRelationship[];
 } & Record<string, any>;
+
+export type JSONGeneratorJhipsterContent = {
+  applicationType: string;
+} & Record<string, any>;
+
+export type JSONRootObject = {
+  entities: string[];
```

Comment:
`entities` is inside `generator-jhipster` too.

Reply:
True, I based my guess on an old version of jh